.github/workflows/release.yml

```diff
@@ -14,68 +22,49 @@ jobs:
-            ${{ runner.os }}-maven-
-            https://github.com/quarkiverse/quarkiverse-template/blob/main/.github/workflows/release.yml
+          cache: 'maven'
+          server-id: ossrh
+          server-username: MAVEN_USERNAME
+          server-password: MAVEN_PASSWORD
+          gpg-private-key: ${{ secrets.GPG_PRIVATE_KEY }}
+          gpg-passphrase: MAVEN_GPG_PASSPHRASE
 
       - name: Configure Git author
         run: |
           git config --local user.email "action@github.com"
           git config --local user.name "GitHub Action"
 
-      - name: maven release ${{steps.metadata.outputs.current-version}}
+      - name: Update latest release version in docs
         run: |
-          gpg --quiet --batch --yes --decrypt --passphrase="${{secrets.GPG_PASSPHRASE}}" --output maven-settings.xml .github/release/maven-settings.xml.gpg
-          git checkout -b release
-          mvn -B release:prepare -Prelease -DreleaseVersion=${{steps.metadata.outputs.current-version}} -DdevelopmentVersion=${{steps.metadata.outputs.next-version}} -s maven-settings.xml
-          git checkout ${{github.base_ref}}
-          git rebase release
-          mvn -B release:perform -Darguments=-DperformRelease -Prelease -s maven-settings.xml
-          git push
-          git push --tags
-
-      - uses: radcortez/milestone-release-action@main
-        name: milestone release
-        with:
-          github-token: ${{secrets.GITHUB_TOKEN}}
-          milestone-title: ${{steps.metadata.outputs.current-version}}
-          milestone-next: ${{steps.metadata.outputs.next-version}}
+          mvn -B -ntp -pl docs -am generate-resources -Denforcer.skip -Dformatter.skip -Dimpsort.skip
+          if ! git diff --quiet docs/modules/ROOT/pages/includes/attributes.adoc; then
+            git add docs/modules/ROOT/pages/includes/attributes.adoc
+            git commit -m "Update the latest release version ${{steps.metadata.outputs.current-version}} in documentation"
+          fi
 
-      - name: generate tck report
+      - name: Maven release ${{steps.metadata.outputs.current-version}}
         run: |
-          cd target/checkout
-          mvn surefire-report:report-only
-          mv target/tck-results.html $GITHUB_WORKSPACE
+          mvn -B release:prepare -Prelease -DreleaseVersion=${{steps.metadata.outputs.current-version}} -DdevelopmentVersion=${{steps.metadata.outputs.next-version}}
+          mvn -B release:perform -Darguments=-DperformRelease -DperformRelease -Prelease
+        env:
+          MAVEN_USERNAME: ${{ secrets.OSSRH_USERNAME }}
+          MAVEN_PASSWORD: ${{ secrets.OSSRH_TOKEN }}
+          MAVEN_GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
 
-      - uses: meeDamian/github-release@2.0
-        name: upload tck asset to release
-        with:
-          token: ${{secrets.GITHUB_TOKEN}}
-          tag: ${{steps.metadata.outputs.current-version}}
-          allow_override: true
-          gzip: false
-          files: tck-results.html
```

Comment:
Can you please readd this piece? This is to publish the TCK results in the release. Thanks!

Reply:
Added back, thanks for reviewing it!